### ISSUE TITLE
[Bug][a11y] The secure connection lock icon doesn't have a label on Talkback

### ISSUE THREAD

**Issue by sv-sdeiac** [mozilla-mobile/fenix#3466]
## Prerequisites
You have Talkback option enabled from your Android settings.

## Steps to reproduce
1. Open a page(ex. Wikipedia.org);
1. Tap on lock icon from the URL bar to be read by the Talkback.

### Expected behavior
The lock icon has a clear and understandable interpretation.

### Actual behavior
The lock icon is described as 'unlabelled button'

### Device information
* Android device: Nexus 6 (Android 7.1.1) and LG G7 fit (Android 8.1)
* Fenix version: 1.0.1924

**Comment by sv-ohorvath** [comment]
duplicate: https://github.com/mozilla-mobile/fenix/issues/1392
